CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ SET(CMAKE_MACOSX_RPATH ON)
 
 SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+# Enable -Wall by default
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
```

Comment:
I would probably use
```
SET(CMAKE_C_FLAGS "-Wall ${CMAKE_C_FLAGS}")
````
instead (ie change the ordering) so that a user can set `CMAKE_C_FLAGS` manually to eg `-Wno-all` and override `-Wall`.

Though I have to to admit broad ignorance of cmake and whether this is the way to do it.

Reply:
Fixed in latest push